plugins/bap/utils/ida.py

```diff
@@ -137,7 +138,7 @@ def dumps(self):
         return ''.join([
             '({:#x} '.format(self.addr),
             ' ({:#x}) '.format(self.fall) if self.fall else '()',
-            '{})'.format(sexps(self.dests))
+            '{})'.format(sexps(self.jmps))
```

Comment:
This is a bug in the original implementation, due to which the normal flow address was being included in the other flows list.